### ISSUE TITLE
Indicate on song selection if there's a chart difference between 17 or 22 frets

### ISSUE THREAD

**Issue by clwalker85** [hmxmilohax/rock-band-3-deluxe#486]
I have no idea how to chart a custom song and whether this information is easily attained, but there are certainly songs that play pro guitar/bass notes all the way up to the 22nd fret for the Squier Telecaster controller, but have a modified chart so that they are playable on the Mustang.

Is there a way to change the pro guitar icon slightly to indicate a song that would have a different pro guitar/bass chart, depending on which controller I'm using? It would be nice to know before I choose the controller I want to use.

**Comment by DarkRTA** [comment]
This is not something we can easily do within the current scope of the mod. We are limited to script edits and have no way of changing the code itself to do this.

**Comment by clwalker85** [comment]
TL;DR - Maybe there's a way to add this 22-fret-needed info to song metadata so it _can_ be done through a script edit. Please excuse any chart ignorance on my part as I brainstorm.

Assuming there are separate charts for 17 fret and 22 fret pro guitar, if there is any difference between the two charts, you could set a true/false flag on the song metadata. This would require adding this extra information song by song, but maybe this can be done programatically through some kind of diff. Since we already add extra information for the pro guitar chart itself on a lot of songs, it didn't seem like a stretch, but I'll be interested to know your thoughts.

**Comment by DarkRTA** [comment]
Gonna reopen this as it may in fact be viable to do this with the new author finder code, although we are likely gonna need some discussion first before we actually choose to pursue this. (if we decide to)

**Comment by DarkRTA** [comment]
Now from my personal point of view, I am currently against doing this using song metadata as we'd essentially be creating yet another piece of metadata™ the community would have to standardize on even though this information is already available in the midi file itself.
